### ISSUE TITLE
MOPs LOPs

### ISSUE THREAD

**Issue by pixel-ninja** [toadstorm/MOPS#332]
Hey,
I just wanted to gauge the interest in bringing the MOPs tools into LOPs. 
Here's a proof of concept with shape falloff and transform modifier working on both xforms and point instancers.

I'd be happy to keep developing the LOP nodes, but not sure of the best process for integrating it here (assuming you're interested in integrating that is).

[MOPs_LOPs_ProofOfConcept.zip](https://github.com/toadstorm/MOPS/files/11391570/MOPs_LOPs_ProofOfConcept.zip)

**Comment by toadstorm** [comment]
I'm definitely interested, just overwhelmed with work and haven't had time to properly sit down and think about integration.
Let me take a look at what you've built here! I can add you as a contributor if you want to submit pull requests.

**Comment by pixel-ninja** [comment]
Fair enough! There's not really any rush, I don't know how many motion designers are really getting stuck into LOPs yet. And even if they are there still ways to leverage the SOP tools in LOPs.

Thanks! I'm running an indie license though so I probably won't be able to push without polluting repo with hdalc files.

**Comment by tweekskratch** [comment]
Hi Matt! I'm not totally into LOPs (yet), but it's on my list of things to
learn :)

Best.
Jason

**Comment by pixel-ninja** [comment]
- Yeah I'll need to do some digging on the transform speed for references. I suspect it either has something to do with either reading the primvars or adding the transform ops.
- I thought that too. The regular Transform LOP seems to have options for flattening and overwriting the xformOpOrder stack so it must be possible but I couldn't find a vex function for it. I might need to hit up SideFX. Failing that it should be possible with python using the usd module but I'm not sure of the speed implications of that.
- Probably yeah. For this initial test I didn't want to deal with nested hdas though. There is also a lot of duplicated vex code that might be better moved to a central mops vex header (i.e. near identical linear falloff code for sops, lop xforms and lop point instances).

Agreed. I tried to completely avoid pivots here. haha.
I'll do some more tinkering and let you know when I have something worth checking out.
